webapp/platform/client/src/client4.ts

```diff
@@ -4187,6 +4198,17 @@ export default class Client4 {
             {method: 'delete', body: JSON.stringify(deletionRequest)},
         );
     }
+
+    // This allows the server to know that a given HTTP request occurred during page load or reconnect.
```

Comment:
Not specifically related to this feature, but if you write this comment using a JSDoc style, it'll actually show that in the tooltip in VS Code when you mouse over the method name
```
/**
 * This allows the server to know...
 */
temporarilySetPageLoadContext = () => {...};
```